`no-undefined-types` report error from `@import`

## Expected behavior

[`no-undefined-types`](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/no-undefined-types.md#readme) reports no errors when using an [`@import` tag](https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#the-jsdoc-import-tag).

## Actual behavior

`no-undefined-types` reports an error for `@import` tag.

## Files

### `package.json`

```json
{
  "name": "testcase",
  "version": "1.0.0",
  "dependencies": {
    "eslint": "9.5.0",
    "eslint-plugin-jsdoc": "48.2.12"
  }
}
```

### `eslint.config.mjs`

```javascript
import jsdoc from "eslint-plugin-jsdoc";

export default [
    {
        plugins: { jsdoc },
        rules: {
            "jsdoc/no-undefined-types": "error"
        }
    }
];
```

### `index.js`

```javascript
/**
 * @import { Linter } from "eslint"
 */
```

## To reproduce

1. `npm install`
2. `npx eslint index.js`

   ```
   /home/regseb/testcase/index.js
     2:1  error  The type 'Linter' is undefined  jsdoc/no-undefined-types

   ✖ 1 problem (1 error, 0 warnings)
   ```

## Environment

- Node version: 20.14.0
- ESLint version: 9.5.0
- `eslint-plugin-jsdoc` version: 48.2.12

--- a/docs/rules/no-undefined-types.md
+++ b/docs/rules/no-undefined-types.md
@@ -776,5 +776,9 @@ function quux(foo) {
 }
 
 quux(0);
+
+/**
+ * @import { Linter } from "eslint"
+ */
 ````
 

--- a/src/rules/noUndefinedTypes.js
+++ b/src/rules/noUndefinedTypes.js
@@ -245,7 +245,7 @@ export default iterateJsdoc(({
   const typeTags = utils.filterTags(({
     tag,
   }) => {
-    return utils.tagMightHaveTypePosition(tag) && (tag !== 'suppress' || settings.mode !== 'closure');
+    return tag !== 'import' && utils.tagMightHaveTypePosition(tag) && (tag !== 'suppress' || settings.mode !== 'closure');
   }).map(tagToParsedType('type'));
 
   const namepathReferencingTags = utils.filterTags(({

--- a/src/tagNames.js
+++ b/src/tagNames.js
@@ -137,6 +137,7 @@ const typeScriptTags = {
   ...jsdocTags,
 
   // https://github.com/microsoft/TypeScript/issues/22160
+  // https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#the-jsdoc-import-tag
   import: [],
 
   // https://www.typescriptlang.org/tsconfig/#stripInternal

--- a/test/rules/assertions/noUndefinedTypes.js
+++ b/test/rules/assertions/noUndefinedTypes.js
@@ -1436,5 +1436,12 @@ export default {
         'no-unused-vars': 'error',
       },
     },
+    {
+      code: `
+        /**
+         * @import { Linter } from "eslint"
+         */
+      `,
+    },
   ],
 };

